services/dask-sidecar/Makefile

```diff
@@ -29,3 +17,30 @@ settings-schema.json: ## [container] dumps json-shcema of this service settings
 		| sed --expression='1,/{/ {/{/!d}' \
 		> $@
 	# Dumped '$(CURDIR)/$@'
+
+.dask-certificates:
+	# create new certificates
+	mkdir --parents $@
+	# Set variables for the key and certificate paths
+	# Run openssl without prompts using the -subj argument to pass subject information
+	key_path="$@/dask-key.pem" && \
```

Comment:
same with the name, if the pattern is well defined, we could even move this to `common.Makefile` and reuse it to create certificates in any of the services under `osparc-simcore/services`

Reply:
I'd like to try first. it's a bit painful to say the least. I also now need to make it work with the ops pendant.
I guess once this goes through and well we can maybe see. but I guess there might be some steps before that